docs/src/API.md

```diff
@@ -0,0 +1,140 @@
+# API
+
+```@meta
+CurrentModule = CLIMAParameters
+```
+
+## Types
+
+```@docs
+AbstractParameterSet
+AbstractEarthParameterSet
+AbstractMicrophysicsParameterSet
+AbstractCloudParameterSet
+AbstractPrecipParameterSet
+AbstractLiquidParameterSet
+AbstractIceParameterSet
+AbstractRainParameterSet
+AbstractSnowParameterSet
+```
+
+## Universal Constants
+
+```@docs
+gas_constant
+light_speed
+h_Planck
+k_Boltzmann
+Stefan
+astro_unit
+```
+
+## Planet
+
+```@docs
+Planet
+Planet.molmass_dryair
+Planet.R_d
+Planet.kappa_d
+Planet.cp_d
+Planet.cv_d
+Planet.ρ_cloud_liq
+Planet.ρ_cloud_ice
+Planet.molmass_water
+Planet.molmass_ratio
+Planet.R_v
+Planet.cp_v
+Planet.cp_l
+Planet.cp_i
+Planet.cv_v
+Planet.cv_l
+Planet.cv_i
+Planet.T_freeze
+Planet.T_min
+Planet.T_max
+Planet.T_icenuc
+Planet.T_triple
+Planet.T_0
+Planet.LH_v0
+Planet.LH_s0
+Planet.LH_f0
+Planet.e_int_v0
+Planet.e_int_i0
+Planet.press_triple
+Planet.ρ_ocean
+Planet.cp_ocean
+Planet.planet_radius
+Planet.day
+Planet.Omega
+Planet.grav
+Planet.year_anom
+Planet.orbit_semimaj
+Planet.TSI
+Planet.MSLP
+Planet.T_surf_ref
+Planet.T_min_ref
+```
+
+## Common
+
+### Sub-grid scale
+
+```@docs
+SubgridScale
+SubgridScale.von_karman_const
+```
+
+## Atmos
+
+```@docs
+Atmos
+```
+
+### Sub-grid scale
+
+```@docs
+Atmos.SubgridScale.C_smag
+Atmos.SubgridScale.C_drag
+Atmos.SubgridScale.inv_Pr_turb
+Atmos.SubgridScale.Prandtl_air
+Atmos.SubgridScale.c_a_KASM
+Atmos.SubgridScale.c_e1_KASM
+Atmos.SubgridScale.c_e2_KASM
+Atmos.SubgridScale.c_1_KASM
+Atmos.SubgridScale.c_2_KASM
+Atmos.SubgridScale.c_3_KASM
+```
+
+### Microphysics
+
+```@docs
+Atmos.Microphysics.n0
```

Comment:
We could also link the relevant ClimateMachine docs somewhere? A huge chunk of the Microphysics docs in the Machine give references to why certain parameters have certain values

Reply:
Hmmm, good question. I think adding a link would be a good idea, yes. I'll add one to the microphysics theory docs.

Reply:
Or not. We could also document here the way the Parameters are meant to be used and keep the theory to the Machine docs. I don't know which is better. 

But maybe a link somewhere up top to the Machine docs would be useful

Reply:
I added a small blurb to the microphysics header. Let me know what you think.